<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BRep</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Edit Mode</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Cube:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="cubeButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../data/icon/icons.qrc">
                  <normaloff>:/cube.png</normaloff>:/cube.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Corner:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QToolButton" name="cornerButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../data/icon/icons.qrc">
                  <normaloff>:/corner.png</normaloff>:/corner.png</iconset>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Cylinder:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QToolButton" name="cylinderButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../data/icon/icons.qrc">
                  <normaloff>:/cylinder.png</normaloff>:/cylinder.png</iconset>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Sphere:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QToolButton" name="sphereButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../data/icon/icons.qrc">
                  <normaloff>:/sphere.png</normaloff>:/sphere.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="sweepingButton">
              <property name="text">
               <string>Sweeping</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Draw Mode</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Line:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="lineButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../data/icon/icons.qrc">
                  <normaloff>:/line.png</normaloff>:/line.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Bezier:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QToolButton" name="bezierButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../data/icon/icons.qrc">
                  <normaloff>:/bezier.png</normaloff>:/bezier.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="doneButton">
              <property name="text">
               <string>Done</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="GLWidget" name="glwidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHerminte">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuHerminte"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionVertexMode"/>
   <addaction name="actionEdgeMode"/>
   <addaction name="actionFaceMode"/>
   <addaction name="actionMeshMode"/>
   <addaction name="separator"/>
   <addaction name="actionR2d2"/>
  </widget>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../data/icon/icons.qrc">
     <normaloff>:/new.png</normaloff>:/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionVertexMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icon/icons.qrc">
     <normaloff>:/vertex.png</normaloff>:/vertex.png</iconset>
   </property>
   <property name="text">
    <string>VertexMode</string>
   </property>
  </action>
  <action name="actionEdgeMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icon/icons.qrc">
     <normaloff>:/edge.png</normaloff>:/edge.png</iconset>
   </property>
   <property name="text">
    <string>EdgeMode</string>
   </property>
  </action>
  <action name="actionFaceMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icon/icons.qrc">
     <normaloff>:/face.png</normaloff>:/face.png</iconset>
   </property>
   <property name="text">
    <string>FaceMode</string>
   </property>
  </action>
  <action name="actionMeshMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icon/icons.qrc">
     <normaloff>:/mesh.png</normaloff>:/mesh.png</iconset>
   </property>
   <property name="text">
    <string>MeshMode</string>
   </property>
  </action>
  <action name="actionR2d2">
   <property name="icon">
    <iconset resource="../../data/icon/icons.qrc">
     <normaloff>:/r2d2.png</normaloff>:/r2d2.png</iconset>
   </property>
   <property name="text">
    <string>R2d2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>GLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../data/icon/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVertexMode</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>vertexmode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdgeMode</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>edgemode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFaceMode</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>facemode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMeshMode</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>meshmode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionR2d2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>r2d2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>zoom()</slot>
  <slot>open()</slot>
  <slot>newModel()</slot>
  <slot>save()</slot>
  <slot>vertexmode()</slot>
  <slot>edgemode()</slot>
  <slot>facemode()</slot>
  <slot>meshmode()</slot>
  <slot>r2d2()</slot>
 </slots>
</ui>
